src/shared/firewall-util-nft.c

```diff
@@ -924,6 +930,29 @@ static int fw_nftables_add_masquerade_internal(
         return sd_nfnl_call_batch(nfnl, &m, 1, NFNL_DEFAULT_TIMEOUT_USECS, NULL);
 }
 
+static int af_to_nfproto(int af) {
+        assert(IN_SET(af, AF_INET, AF_INET6));
+
+        switch (af) {
+        case AF_INET:
+                return NFPROTO_IPV4;
+        case AF_INET6:
+                return NFPROTO_IPV6;
+        default:
+                assert_not_reached();
+        }
+}
+
+static int fw_nftables_add_masquerade_internal(
+                sd_netlink *nfnl,
+                bool add,
+                int af,
+                const union in_addr_union *source,
+                unsigned int source_prefixlen) {
+        return nft_set_element_modify_in_addr(nfnl, add, af_to_nfproto(af), af, NFT_SYSTEMD_TABLE_NAME, NFT_SYSTEMD_MASQ_SET_NAME,
+                                              source, source_prefixlen);
+}
```

Comment:
Please drop such an one line static wrapper. This is only used twice in the below.

Reply:
ok